frontend/test/metabase/scenarios/models/model-actions.cy.spec.js

```diff
@@ -4,17 +4,19 @@ import {
   popover,
   restore,
 } from "__support__/e2e/helpers";
-import { SAMPLE_DB_TABLES } from "__support__/e2e/cypress_data";
+
 import { createMockActionParameter } from "metabase-types/api/mocks";
 
 const PG_DB_ID = 2;
+const PG_ORDERS_TABLE_ID = 9;
 
 const SAMPLE_ORDERS_MODEL = {
   name: "Order",
   dataset: true,
   display: "table",
+  database: PG_DB_ID,
   query: {
-    "source-table": SAMPLE_DB_TABLES.STATIC_ORDERS_ID,
+    "source-table": PG_ORDERS_TABLE_ID,
   },
 };
```

Comment:
Turns out the test suite had a model based on a sample database and an action based on the Postgres QA DB 😛